src/operators.jl

```diff
@@ -66,3 +66,61 @@ end
 function Base.:(==)(a::SpinMonomial, b::SpinMonomial)
     return a.variables == b.variables
 end
+
+function Base.:(==)(term1::SpinTerm{T}, term2::SpinTerm{T}) where T
```

Comment:
This function is not needed, it is already implemented in
https://github.com/JuliaAlgebra/MultivariatePolynomials.jl/blob/725642e2d159fc71ad34f119e932c0715973d017/src/comparison.jl#L43-L51